lib/generic/context_runner.rb

```diff
@@ -0,0 +1,103 @@
+module Synthea
```

Comment:
Is there any way to simplify the code in the ContextRunner / Context / Package trio? It works but it feels like it's more complicated than it has to be. Maybe there's no clean way to do it without carving out massive changes but I'm going to keep thinking about that.